strawberryfields/configuration.py

```diff
@@ -222,6 +295,81 @@ def parse_environment_variable(key, value):
     return value
 
 
+def active_configs(filename="config.toml"):
+    """Prints the filepaths for existing configuration files to the standard
+    output and marks the one that is active.
+
+    This function relies on the precedence ordering of directories to check
+    when marking the active configuration.
+
+    Args:
+        filename (str): the name of the configuration files to look for
+    """
+    active_configs_list = get_active_configs(filename)
+    print_active_configs(active_configs_list, filename)
+    print_directories_checked()
+
+
+def get_active_configs(filename="config.toml"):
```

Comment:
How about `get_available_config_paths`? Also, this function is missing an initial docstring summary!

Reply:
Good catch, thanks!

Something along (?):
`Get the paths for the configuration files available in Strawberry Fields.`